backend/pkg/api/data_access/vdb_summary.go

```diff
@@ -732,62 +738,138 @@ func (d *DataAccessService) GetValidatorDashboardGroupSummary(ctx context.Contex
 	return ret, nil
 }
 
-func (d *DataAccessService) internal_getElClAPR(ctx context.Context, validators []t.VDBValidator, days int) (elIncome decimal.Decimal, elAPR float64, clIncome decimal.Decimal, clAPR float64, err error) {
```

Comment:
The reason I had to change the whole method from a validator list `validators` to the `dashboardId` (+group filter) is because I could not change the query for clickhouse to
`WHERE validator_index IN ($1)`

For a large dashboard, e.g. 250,000 validators, you get a "Max query size exceeded" error.